### ISSUE TITLE
High Battery Drain on iOS

### ISSUE THREAD

**Issue by ScriptInfra** [mullvad/mullvadvpn-app#4858]
## Issue report

Operating system: iOS 16.5.1
App version: 2023.2

According to iOS battery life consumption graph, the Mullvad app is consuming a tremendous amount of battery. This high level of battery usage is also felt in real time. The battery is draining very quickly.

![IMG_5567](https://github.com/mullvad/mullvadvpn-app/assets/63025323/5369af76-5b0a-4c59-b2f0-f031ca8b24b1)

![IMG_5566](https://github.com/mullvad/mullvadvpn-app/assets/63025323/8ac070b2-1c2c-4456-bc54-6ef908e2592a)

![IMG_5565](https://github.com/mullvad/mullvadvpn-app/assets/63025323/1010047d-871b-43af-8542-e73d9e0a9a2a)

**Comment by ScriptInfra** [comment]
This issue can probably be closed. Updating to iOS 16.6 has resolved all battery life issues.

**Comment by faern** [comment]
Thats great to hear! Battery life issues are so hard to track down because they are hard to reproduce and take a long time to test. So I'm glad it's resolved for you

**Comment by acrostich** [comment]
I noticed a big change in battery life by switching all my mail accounts from Push to Manual. My phone didn't last a full day and now does. The only account I've left on Push is iCloud (which somehow is in the Mail/Accounts/Fetch New Data setting screen).

I suspect that Push on some or all of the IMAP accounts was pushed too often even though I don't receive that much mail which in turn activated Mullvad and used more battery.

I have set the Fetch checks to Hourly and I've noticed in the morning that these hourly checks do not appear to have happened. Not sure if it's a bug in mail.app or iOS, but that would also reduce battery usage of both mail.app and mullvad even further.

So for anyone having large battery drains it is worth checking their Mail settings and considering switching to Manual.It might also indicate an issue if there are some timeouts in Mullvad that means the Fetch prevents it from going to sleep (at times).
